plugins/scaffolder-react/src/next/components/ReviewState/ReviewState.tsx

```diff
@@ -28,14 +28,47 @@ export type ReviewStateProps = {
   formState: JsonObject;
 };
 
+function flattenObject(
```

Comment:
It worths to move this logic to some separate file, `util.ts` or `flatten.ts` and cover it separately with a unit test.
The less logic in UI component is better, easier to comprehend what is going on a representation side.

Reply:
Sure. Any suggestions where a good place would be to place this file?

Reply:
I would keep this file in the same folder as it still belongs to this component only

Reply:
Done!